Px.Utils/Validation/Enums.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace PxUtils.Validation
+{
+    /// <summary>
+    /// Defines the levels of validation feedback. These levels can be used to categorize feedback items by severity.
+    /// </summary>
+    public enum ValidationFeedbackLevel
+    {
+        Warning,
```

Comment:
I suggest adding fixed numeric values to these, so they have explicit order. Warning = 0 and Error = 1 for example.